### PR TITLE
3.0 The integration of general json and rest

### PR DESCRIPTION
**Pull request by yamicro**
The integration of general json and rest in 3.0

### CODE REVIEW

**Review by EnableAsync — CHANGES_REQUESTED** [review]
.DS_Store can be removed.

File: general/jsonrpc/go-client/conf/client.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ connect_timeout : "3s"
 
 # application config
 application:
-  organization : "ikurento.com"
+  organization : "org.appache.dubbo"
```
Comment (LaurenceLiZhixin) [inline general/jsonrpc/go-client/conf/client.yml:11]: apache

File: general/jsonrpc/go-server/tests/integration/userprovider_test.go [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package integration
+
+import (
+	"github.com/apache/dubbo-go-samples/general/jsonrpc/go-client/pkg"
```
Comment (LaurenceLiZhixin) [inline general/jsonrpc/go-server/tests/integration/userprovider_test.go:23]: change to 3 blocks

File: general/jsonrpc/go-server/tests/integration/userprovider_test.go [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+	_ "dubbo.apache.org/dubbo-go/v3/protocol/jsonrpc"
+	_ "dubbo.apache.org/dubbo-go/v3/registry/protocol"
+
+	_ "dubbo.apache.org/dubbo-go/v3/filter/filter_impl"
+
+	_ "dubbo.apache.org/dubbo-go/v3/cluster/cluster_impl"
+	_ "dubbo.apache.org/dubbo-go/v3/cluster/loadbalance"
+	_ "dubbo.apache.org/dubbo-go/v3/registry/zookeeper"
+)
+var 	survivalTimeout int = 10e9
```
Comment (LaurenceLiZhixin) [inline general/jsonrpc/go-server/tests/integration/userprovider_test.go:53]: can you fixs these space?

File: general/rest/go-server/tests/integration/userprovider_test.go [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+import (
+	"dubbo.apache.org/dubbo-go/v3/common/logger"
+	_ "dubbo.apache.org/dubbo-go/v3/common/proxy/proxy_factory"
+	_ "dubbo.apache.org/dubbo-go/v3/filter/filter_impl"
+	_ "dubbo.apache.org/dubbo-go/v3/protocol/rest"
+	_ "dubbo.apache.org/dubbo-go/v3/registry/protocol"
+
+	_ "dubbo.apache.org/dubbo-go/v3/cluster/cluster_impl"
+	_ "dubbo.apache.org/dubbo-go/v3/cluster/loadbalance"
+	_ "dubbo.apache.org/dubbo-go/v3/registry/zookeeper"
```
Comment (LaurenceLiZhixin) [inline general/rest/go-server/tests/integration/userprovider_test.go:50]: Also, space and import blocks

**Comment by LaurenceLiZhixin** [comment]
Pls fix you ci check.

**Pull request closed without merging**